src/js/Content/Features/Store/Funds/FCustomGiftcardAndWallet.js

```diff
@@ -16,9 +22,14 @@ export default class FCustomGiftcardAndWallet extends Feature {
 
         const currency = CurrencyManager.fromType(CurrencyManager.storeCurrency);
         const minValue = currency.valueOf(price);
-        const step = 10 ** -currency.format.decimalPlaces;
 
-        let input = `<input type="number" id="es_custom_money_amount" class="es_text_input money" min="${minValue}" step="${step}" value="${minValue}">`;
+        const submitel = newel.querySelector("a");
```

Comment:
For the record, this is the actual fix: Steam removed the redundant `btn_medium` class from giftcards. There's only one clickable link within each funds/giftcard element, so a simple selector should be enough.